python/arcticdb/version_store/_normalization.py

```diff
@@ -937,6 +938,31 @@ def _packb(*args, **kwargs):
     return packb(*args, **kwargs)
 
 
+def allocate_buffer(max_size: int , attempts: int = 4, divisor: int = 2):
```

Comment:
This combination of parameters means that the smallest attempted allocation will be 500MB, I would add a couple more attempts